package/tests/testthat/_snaps/pkct01/markdown-snaps.md

```diff
@@ -8,27 +8,96 @@
+                                               n    LTRs/BLQs    Mean       SD     CV (%) Mean   Geometric Mean   CV % Geometric Mean   Median   Minimum   Maximum
+      ————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+      A: Drug X                                                                                                                                                   
+        Day 1                                                                                                                                                     
+          0                                   134      134       0        0            NE            0                    NE             0        0         0     
+          0.5                                 134        0      12.6      1.51        12.0          12.5                  12.2          12.6      9.72     15.6   
+          1                                   134        0      16.2      1.63        10.0          16.1                  10.1          16.2     12.6      19.9   
+          1.5                                 134        0      15.6      1.46         9.3          15.6                   9.3          15.5     12.3      19     
+          2                                   134        0      13.4      1.35        10.1          13.4                  10.0          13.3     10.8      16.5   
+          3                                   134        0       8.47     1.25        14.7           8.38                 15.0           8.4      5.88     10.9   
+          4                                   134        0       4.79     1.02        21.2           4.69                 22.0           4.79     2.7       7.09  
+          8                                   134        0       0.348    0.18        51.7           0.303                58.4           0.318    0.076     0.866 
+          12                                  134        0       0.0224   0.0189      84.6           0.0156              111.6           0.017    0.002     0.083 
+        Day 2                                                                                                                                                     
+          24                                  134      134       0        0            NE            0                    NE             0        0         0     
+      ————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
       
       NE: Not Estimable
 
+# result_v2
+
+    Code
+      print(data_snap[[i]])
+    Output
+      Summary of PK Concentrations by Nominal Time and Treatment: PK Evaluable
+      Protocol: xxxxx
+      Analyte:  Plasma Drug X
+      Treatment: A: Drug X
```

Comment:
@qiliu1013 can you check if this is correct (imputation rule 1/3)

@shajoezhu fyi

Reply:
Looks good! Thanks.